### ISSUE TITLE
[Discussion] Data usage

### ISSUE THREAD

**Issue by TanukiSharp** [AthenaADP/MHW-ASS#2]
Hello,

I wanted to contact you in private since for sure nobody will have any interest in this conversation, and absolutely useless for it to remain as a closed issue, but I couldn't find any other way to reach you. (I do not use Facebook, and will probably never)
 
I'm also developing an armor set search, for MHW and for Window (PC) only.

I guess you are aware how getting data (and the correct one) is hard. I found a lot of different data sources for my application, but none of them are as accurate and as complete in term of what's need for an armor set search application as yours. Therefore, I wanted to tell you that I'm using your data.

In the beginning I thought it's mean to just steal your data, so the data loader in my application was requesting users to download ASS first, and indicate their own ASS path for my application to work, so I didn't have to ship your data with my application. Many users (very few users but most of those few users) complained this was a pain, so switching to another data source got higher priority in my TODO list. I tried different sources, and even contributed to other data sources, but it was still far from being as satisfying as using your data. Then I discovered your GitHub account and saw it is MIT. Since my repository is also MIT, I think it's legal for me to use you data. I'm not complaining CSV is a bad format, but I opted for converting it to JSON offline with a tool I developed, so the loader in the application can be much smaller (less code), much faster, and code is much more maintainable.

So I'm hereby asking you if this is fine with you that I use ASS' data. I would understand you'd refuse, I'd be in trouble but then I would respect your choice and find another way.

You can find my project here: https://github.com/TanukiSharp/MHArmory

I'm grateful I can use your your data, and thus to thank you for this hard work, I would like to help you by contributing back to ASS if I can. I know WinForms and .NET quite a lot, I'm familiar with C++, not much with C++ Common Language Infrastructure, but I can learn if need be.

Sorry for this long conversation, and thank you or your time.

**Comment by AthenaADP** [comment]
You are welcome to use any and all code and/or data here or otherwise distributed in the ASS Data folder. I'd appreciate a mention/attribution though.

And if you find any mistakes please let me know so I can update things. Cheers.

**Comment by TanukiSharp** [comment]
Thank you very much for this kind answer :)

As for the code, I never had a look at your code, since I wanted to implement mine 100% by myself for the challenge, having my own bugs and my own experience along the way :)

As for the data, I found only one mistake and already made a pull request (https://github.com/AthenaADP/MHW-ASS/pull/1). I saw lately you fixed the Lavasioth Greaves alpha that I never spotted before.

As for the mention, it's already on the GitHub page of my application since the beginning, but I will mention that I explicitly use your data so there cannot be any misunderstanding.

Thanks again :)
